packages/truffle-compile/index.js

```diff
@@ -367,6 +365,9 @@ compile.necessary = function(options, callback) {
     if (err) return callback(err);
 
     if (updated.length === 0 && options.quiet !== true) {
+      if (options.eventManager) {
+        options.eventManager.emitEvent("compile:nothingToCompile");
```

Comment:
Looks like if nothing is updated AND there are no contracts to compile the `nothingToCompile` event gets emitted/reported twice.

<img width="399" alt="screen shot 2019-02-26 at 2 48 04 pm" src="https://user-images.githubusercontent.com/7537712/53452220-898c4780-39d5-11e9-994a-03c2505677c1.png">

Reply:
(Also liked the ========= line stuff in the earlier code)